src/azure-cli/setup.py

```diff
@@ -52,62 +52,62 @@
-    'azure-cli-botservice',
-    'azure-cli-cdn',
-    'azure-cli-cloud',
-    'azure-cli-cognitiveservices',
-    'azure-cli-configure',
-    'azure-cli-consumption',
-    'azure-cli-container',
-    'azure-cli-cosmosdb',
-    'azure-cli-dla',
-    'azure-cli-dls',
-    'azure-cli-dms',
-    'azure-cli-eventgrid',
-    'azure-cli-eventhubs',
-    'azure-cli-extension',
-    'azure-cli-feedback',
-    'azure-cli-find',
-    'azure-cli-hdinsight',
-    'azure-cli-interactive',
-    'azure-cli-iot',
-    'azure-cli-iotcentral',
-    'azure-cli-keyvault',
-    'azure-cli-kusto',
-    'azure-cli-lab',
-    'azure-cli-maps',
-    'azure-cli-monitor',
-    'azure-cli-network',
-    'azure-cli-policyinsights',
-    'azure-cli-profile',
-    'azure-cli-rdbms',
-    'azure-cli-redis',
-    'azure-cli-relay',
-    'azure-cli-reservations',
-    'azure-cli-resource',
-    'azure-cli-role',
-    'azure-cli-search',
-    'azure-cli-security',
-    'azure-cli-servicebus',
-    'azure-cli-servicefabric',
-    'azure-cli-signalr',
-    'azure-cli-sql',
-    'azure-cli-sqlvm',
-    'azure-cli-storage',
-    'azure-cli-vm'
+    'azure-cli-acr==2.2.1',
+    'azure-cli-acs==2.3.18',
+    'azure-cli-advisor==2.0.0',
+    'azure-cli-ams==0.4.2',
+    'azure-cli-appservice==0.2.14',
+    'azure-cli-backup==1.2.1',
+    'azure-cli-batch==4.0.0',
+    'azure-cli-batchai==0.4.7',
+    'azure-cli-billing==0.2.0',
+    'azure-cli-botservice==0.1.7',
+    'azure-cli-cdn==0.2.0',
+    'azure-cli-cloud==2.1.0',
+    'azure-cli-cognitiveservices==0.2.4',
+    'azure-cli-command_modules-nspkg==2.0.2',
+    'azure-cli-configure==2.0.20',
+    'azure-cli-consumption==0.4.2',
+    'azure-cli-container==0.3.14',
+    'azure-cli-core==2.0.59',
+    'azure-cli-cosmosdb==0.2.8',
+    'azure-cli-dla==0.2.4',
+    'azure-cli-dls==0.1.8',
+    'azure-cli-dms==0.1.2',
+    'azure-cli-eventgrid==0.2.1',
+    'azure-cli-eventhubs==0.3.3',
+    'azure-cli-extension==0.2.3',
+    'azure-cli-feedback==2.1.4',
+    'azure-cli-find==0.2.13',
+    'azure-cli-hdinsight==0.3.1',
+    'azure-cli-interactive==0.4.1',
+    'azure-cli-iot==0.3.6',
+    'azure-cli-iotcentral==0.1.6',
+    'azure-cli-keyvault==2.2.12',
+    'azure-cli-kusto==0.2.0',
+    'azure-cli-lab==0.1.5',
+    'azure-cli-maps==0.3.3',
+    'azure-cli-monitor==0.2.10',
+    'azure-cli-network==2.3.3',
+    'azure-cli-nspkg==3.0.3',
+    'azure-cli-policyinsights==0.1.1',
+    'azure-cli-profile==2.1.3',
+    'azure-cli-rdbms==0.3.7',
+    'azure-cli-redis==0.4.1',
+    'azure-cli-relay==0.1.3',
+    'azure-cli-reservations==0.4.1',
+    'azure-cli-resource==2.1.11',
+    'azure-cli-role==2.4.1',
+    'azure-cli-search==0.1.1',
+    'azure-cli-security==0.1.0',
+    'azure-cli-servicebus==0.3.3',
+    'azure-cli-servicefabric==0.1.13',
+    'azure-cli-signalr==1.0.0',
+    'azure-cli-sql==2.1.9',
+    'azure-cli-sqlvm==0.1.0',
+    'azure-cli-storage==2.3.1',
+    'azure-cli-telemetry==1.0.1',
+    'azure-cli-vm==2.2.16'
```

Comment:
You have to unpin the versions in the dev branch or dev installation will break.